src/server/pfs/server/driver_fsck.go

```diff
@@ -365,22 +366,28 @@ func (d *driver) fsck(ctx context.Context, fix bool, cb func(*pfs.FsckResponse)
 	branchInfos := make(map[string]*pfs.BranchInfo)
 	commitInfos := make(map[string]*pfs.CommitInfo)
 	repoInfos := make(map[string]*pfs.RepoInfo)
-	repoInfo := &pfs.RepoInfo{}
-	if err := d.repos.ReadOnly(ctx).List(repoInfo, col.DefaultOptions(), func(string) error {
```

Comment:
Can we have a brief explanation here for the original intent for read only, and why we are switching to transactional.

Reply:
I think we discussed this at the team meeting on 8/8/2023